MarvinLee_Portfolio_FEND/index.html

```diff
@@ -21,32 +21,38 @@ <h1 class="name">Marvin Lee</h1>
       </div>
     </header>
 
-    <main>
-      <div class="mainImage">
-          <img class="mainCode" src="images/main.jpg" alt="main image">
+    <main class='main-content'>
+      <div class="main-image-container">
+          <img class="theme-image" src="images/main.jpg" alt="main image">
       </div>
 
       <section class="featured_work">
         <h2 class="third_title">Featured Work</h2>
-        <figure class="featured_images_left">
+        <figure class="featured_image">
             <img class="images_left" src="images/appify.png" alt="appify">
               <figcaption class="featured_fig">
-                <h2><a href="#">APPIFY</a></h2>
-                <p class="featured_phr"><a href="#">https://github.com/udacity/Appify</a></p>
+                <a href="#">
+                  <h2>APPIFY</h2>
+                  <p class="project-info">My first project</p>
+                </a>
               </figcaption>
         </figure>
-        <figure class="featured_images_center">
+        <figure class="featured_image">
             <img class="images_center" src="images/sunflower.png" alt="appify">
               <figcaption class="featured_fig">
-                <h2><a href="#">SUNFLOWER</a></h2>
-                <p class="featured_phr"><a href="#">https://github.com/udacity/Sunflower</a></p>

```

Comment:
You do not need to include actual links. Instead wrap h2 and short description in <a> tags.